Engine/Source/LineComponent.cpp

```diff
@@ -0,0 +1,103 @@
+#include "LineComponent.h"
+#include "Application.h"
+#include "ModuleOpenGL.h"
+#include "ModuleResource.h"
+#include "Application.h"
+#include "ModuleOpenGL.h"
+#include "ModuleCamera.h"
+#include "CameraComponent.h"
+#include "ResourceTexture.h"
+#include "ModuleResource.h"
+#include "glew.h"
+#include "MathGeoLib.h"
+#include "Timer.h"
+#include "Trail.h"
```

Comment:
#include "MathGeoLib.h"
#include "Timer.h"
#include "glew.h"
#include "ModuleResource.h"
#include "ResourceTexture.h"
#include "CameraComponent.h"
#include "ModuleCamera.h"
are not needed

#include "ModuleResource.h"
#include "ModuleOpenGL.h"
#include "Application.h"
are duplicated